aws.yaml

```diff
@@ -1,3 +1,18 @@
+- active: false
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 8.0.2
```

Comment:
I think it should be 8.0.3-dev https://github.com/giantswarm/aws-operator/blob/4f75376aecaeb4392f944731883ef2ee8925cf25/pkg/project/project.go#L4

Reply:
Thanks I missed that. https://github.com/giantswarm/releases/pull/69 will fix it.